src/deluge/storage/flash_storage.cpp

```diff
@@ -636,6 +648,7 @@ void readSettings() {
 		gui::menu_item::fillColourMenu.value = gui::menu_item::Colour::AMBER;
 		gui::menu_item::onceColourMenu.value = gui::menu_item::Colour::MAGENTA;
 	}
+	defaultSliceMode == buffer[167];
 }
```

Comment:
now this gies me an "cannot convert 'std::span<unsigned char>::element_type' {aka 'unsigned char'} to 'SampleRepeatMode' in assignment"

Reply:
to fix this error:

defaultSliceMode = static_cast<SampleRepeatMode>(buffer[167]);

Reply:
sorry my suggestion above didn't show properly, just wrapped it in code tags so you can see what's needed

Reply:
uff whats that even? like a conversion?

Reply:
> uff whats that even? like a conversion?

Yep it's converting the value that's in the buffer to type SampleRepeatMode so that it can be written into the defaultSliceMode variable of type SampleRepeatMode

Reply:
gotcha. would have never ever figured this one out